src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -128,6 +128,9 @@ export class CalcitePopover {
   /** Text for close button. */
   @Prop() intlClose = TEXT.close;
 
+  /** Appearance of the corners (rounded or flat) */
+  @Prop({ reflect: true }) cornerAppearance: "flat" | "round" = "flat";
```

Comment:
Do we need this prop? How would a user know when to use one or the other?

Reply:
Resolved by: https://github.com/Esri/calcite-components/commit/ef3ac11e06c7196ff4db7615f802e6d79bae0a82